### ISSUE TITLE
KDE6 Build error

### ISSUE THREAD

**Issue by rejedai** [kupiqu/SierraBreezeEnhanced#128]
I tried building the package under the beta version of KDE6, but got an error:

```
CMake Error: The INTERFACE_QT_MAJOR_VERSION property of "Qt6::Gui" does
not agree with the value of QT_MAJOR_VERSION already determined
for "sierrabreezeenhanced".

-- Generating done (0.0s)
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

**Comment by serhumanos** [comment]
is there any support for plasma 6 ?

**Comment by kupiqu** [comment]
Sorry, I am very busy at the moment. Please consider to contribute submitting a pull request.

**Comment by OzzyHelix** [comment]
I'm not a developer so I can't provide a pull request I just hope someone can update this to work

**Comment by chiyuki0325** [comment]
#130

**Comment by OzzyHelix** [comment]
I can confirm that building their branch works but with some errors it took a couple tries to get it to compile this is the terminal output that contains the errors 
```
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeexceptiondialog.cpp: In lambda function:
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeexceptiondialog.cpp:66:98: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
   66 |         connect( m_ui.gradientOverrideLabelSpinBox, QOverload<int>::of(&QSpinBox::valueChanged), [=](int /*i*/){updateChanged();} );
      |                                                                                                  ^
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeexceptiondialog.cpp:66:98: note: add explicit ‘this’ or ‘*this’ capture
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeexceptiondialog.cpp: In lambda function:
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeexceptiondialog.cpp:69:97: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
   69 |         connect( m_ui.opacityOverrideLabelSpinBox, QOverload<int>::of(&QSpinBox::valueChanged), [=](int /*i*/){updateChanged();} );
      |                                                                                                 ^
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeexceptiondialog.cpp:69:97: note: add explicit ‘this’ or ‘*this’ capture
make[1]: *** [CMakeFiles/Makefile2:667: config/CMakeFiles/kcm_sierrabreezeenhanceddecoration.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeconfigwidget.cpp: In lambda function:
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeconfigwidget.cpp:51:83: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
   51 |         connect( m_ui.buttonSpacing, QOverload<int>::of(&QSpinBox::valueChanged), [=](int /*i*/){updateChanged();} );
      |                                                                                   ^
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeconfigwidget.cpp:51:83: note: add explicit ‘this’ or ‘*this’ capture
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeconfigwidget.cpp: In lambda function:
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeconfigwidget.cpp:55:89: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
   55 |         connect( m_ui.cornerRadiusSpinBox, QOverload<int>::of(&QSpinBox::valueChanged), [=](int /*i*/){updateChanged();} );
      |                                                                                         ^
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeconfigwidget.cpp:55:89: note: add explicit ‘this’ or ‘*this’ capture
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeconfigwidget.cpp: In lambda function:
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeconfigwidget.cpp:61:84: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
   61 |         connect( m_ui.opacitySpinBox, QOverload<int>::of(&QSpinBox::valueChanged), [=](int /*i*/){updateChanged();} );
      |                                                                                    ^
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeconfigwidget.cpp:61:84: note: add explicit ‘this’ or ‘*this’ capture
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeconfigwidget.cpp: In lambda function:
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeconfigwidget.cpp:62:85: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
   62 |         connect( m_ui.gradientSpinBox, QOverload<int>::of(&QSpinBox::valueChanged), [=](int /*i*/){updateChanged();} );
      |                                                                                     ^
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeconfigwidget.cpp:62:85: note: add explicit ‘this’ or ‘*this’ capture
/home/ozzy/Documents/development/SierraBreezeEnhanced/breezedecoration.cpp: In member function ‘virtual void Breeze::Decoration::hoverMoveEvent(QHoverEvent*)’:
/home/ozzy/Documents/development/SierraBreezeEnhanced/breezedecoration.cpp:358:86: warning: ‘QPointF QHoverEvent::posF() const’ is deprecated: Use position() [-Wdeprecated-declarations]
  358 |             const bool groupContains = m_leftButtons->geometry().contains(event->posF()) || m_rightButtons->geometry().contains(event->posF());
      |                                                                           ~~~~~~~~~~~^~
In file included from /usr/include/qt6/QtGui/QMouseEvent:1,
                 from /home/ozzy/Documents/development/SierraBreezeEnhanced/breezesizegrip.h:26,
                 from /home/ozzy/Documents/development/SierraBreezeEnhanced/breezedecoration.cpp:31:
/usr/include/qt6/QtGui/qevent.h:265:20: note: declared here
  265 |     inline QPointF posF() const { return position(); }
      |                    ^~~~
/home/ozzy/Documents/development/SierraBreezeEnhanced/breezedecoration.cpp:358:140: warning: ‘QPointF QHoverEvent::posF() const’ is deprecated: Use position() [-Wdeprecated-declarations]
  358 |             const bool groupContains = m_leftButtons->geometry().contains(event->posF()) || m_rightButtons->geometry().contains(event->posF());
      |                                                                                                                                 ~~~~~~~~~~~^~
/usr/include/qt6/QtGui/qevent.h:265:20: note: declared here
  265 |     inline QPointF posF() const { return position(); }
      |                    ^~~~
[ 93%] Linking CXX shared module bin/sierrabreezeenhanced.so
[ 95%] Built target sierrabreezeenhanced
make: *** [Makefile:136: all] Error 2
❯ make -j24
[  0%] Built target sierrabreezeenhancedcommon6_autogen_timestamp_deps
[  0%] Built target sierrabreezeenhanced-kcm-desktop-gen
[  0%] Built target kcm_sierrabreezeenhanceddecoration-kcm-desktop-gen
[  2%] Built target sierrabreezeenhancedcommon6_autogen
[  2%] Built target kcm_sierrabreezeenhanceddecoration_autogen_timestamp_deps
[ 11%] Built target sierrabreezeenhancedcommon6
[ 13%] Built target kcm_sierrabreezeenhanceddecoration_autogen
[ 13%] Built target sierrabreezeenhanced_autogen_timestamp_deps
[ 16%] Building CXX object config/CMakeFiles/kcm_sierrabreezeenhanceddecoration.dir/kcm_sierrabreezeenhanceddecoration_autogen/mocs_compilation.cpp.o
[ 18%] Building CXX object config/CMakeFiles/kcm_sierrabreezeenhanceddecoration.dir/breezedetectwidget.cpp.o
[ 20%] Building CXX object config/CMakeFiles/kcm_sierrabreezeenhanceddecoration.dir/breezeexceptiondialog.cpp.o
[ 23%] Built target sierrabreezeenhanced_autogen
[ 69%] Built target sierrabreezeenhanced
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeexceptiondialog.cpp: In lambda function:
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeexceptiondialog.cpp:66:98: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
   66 |         connect( m_ui.gradientOverrideLabelSpinBox, QOverload<int>::of(&QSpinBox::valueChanged), [=](int /*i*/){updateChanged();} );
      |                                                                                                  ^
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeexceptiondialog.cpp:66:98: note: add explicit ‘this’ or ‘*this’ capture
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeexceptiondialog.cpp: In lambda function:
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeexceptiondialog.cpp:69:97: warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
   69 |         connect( m_ui.opacityOverrideLabelSpinBox, QOverload<int>::of(&QSpinBox::valueChanged), [=](int /*i*/){updateChanged();} );
      |                                                                                                 ^
/home/ozzy/Documents/development/SierraBreezeEnhanced/config/breezeexceptiondialog.cpp:69:97: note: add explicit ‘this’ or ‘*this’ capture
[ 72%] Linking CXX shared module ../bin/org.kde.kdecoration2.kcm/kcm_sierrabreezeenhanceddecoration.so
[100%] Built target kcm_sierrabreezeenhanceddecoration
❯ sudo make install
[  0%] Built target sierrabreezeenhancedcommon6_autogen_timestamp_deps
[  2%] Built target sierrabreezeenhancedcommon6_autogen
[ 11%] Built target sierrabreezeenhancedcommon6
[ 11%] Built target sierrabreezeenhanced-kcm-desktop-gen
[ 11%] Built target sierrabreezeenhanced_autogen_timestamp_deps
[ 13%] Built target sierrabreezeenhanced_autogen
[ 60%] Built target sierrabreezeenhanced
[ 60%] Built target kcm_sierrabreezeenhanceddecoration-kcm-desktop-gen
[ 60%] Built target kcm_sierrabreezeenhanceddecoration_autogen_timestamp_deps
[ 62%] Built target kcm_sierrabreezeenhanceddecoration_autogen
[100%] Built target kcm_sierrabreezeenhanceddecoration
Install the project...
-- Install configuration: "Release"
-- Installing: /usr/lib/qt6/plugins/org.kde.kdecoration2/sierrabreezeenhanced.so
-- Set non-toolchain portion of runtime path of "/usr/lib/qt6/plugins/org.kde.kdecoration2/sierrabreezeenhanced.so" to ""
-- Installing: /usr/share/applications/sierrabreezeenhanced.desktop
-- Installing: /usr/lib/libsierrabreezeenhancedcommon6.so.1.3.3
-- Installing: /usr/lib/libsierrabreezeenhancedcommon6.so.0
-- Installing: /usr/lib/qt6/plugins/org.kde.kdecoration2.kcm/kcm_sierrabreezeenhanceddecoration.so
-- Installing: /usr/share/applications/kcm_sierrabreezeenhanceddecoration.desktop
```

**Comment by kupiqu** [comment]
Fixed in Version 2.0.0
